pyspecdata/lmfitdata.py

```diff
@@ -105,9 +105,9 @@ def functional_form(self, this_expr):
             modules=[{"ImmutableMatrix": np.ndarray}, "numpy", "scipy"],
         )
 
-        self.pars = Parameters()
+        self.guess_coeff = Parameters()
         for this_name in self.parameter_names:
-            self.pars.add(this_name)
+            self.guess_coeff.add(this_name)
```

Comment:
throughout, I changed the name of `self.pars` to `self.guess_coeff` since that's a better name for what it is, and because I think this was causing you confusion.